jakartaee9/src/ejb-jar_4_0.xsds

```diff
@@ -1973,9 +1973,8 @@
               superclass around-timeout methods.
 	    - an optional declaration of the bean's environment
 	      entries
-	    - an optional declaration of the bean's EJB references
-	    - an optional declaration of the bean's local EJB
-	      references
+	    - an optional declaration of the bean's references
```

Comment:
Should be "the bean's enterprise bean references".  This is about a bean having references to other beans